client/session.go

```diff
@@ -543,19 +542,18 @@ func valuesToBytes(dataTypes []TSDataType, values []interface{}) ([]byte, error)
 				return nil, fmt.Errorf("values[%d] %v(%v) must be float32", i, v, reflect.TypeOf(v))
 			}
 		case DOUBLE:
-			switch v.(type) {
+			switch v := v.(type) {
```

Comment:
It's better to not shadow the variable `v`. `v`'s type is `interface{}`. I suggest that we use a different variable name like `dv` instead of `v`.